src/content/1.7/modules/sell/techvalidation-keysteps.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Technical validation - Key steps
+---
+
+# Technical Validation : Key steps
+
+Have you ever wondered how the PrestaShop Addons Team treats the validation ?
+
+We are going to show you how to win time so you can pretty much "validate" your own module before submitting it on the Marketplace Addons.
+First you might want to read the [Technical Tools]({{<ref "1.7/modules/sell/technical-tools" >}}) and the [Good Practices]({{<ref "1.7/modules/creation/good-practices" >}}). 
+
+### 1. The Basics
+
+As most of you know, there are 3 types of submittions : New, Update min, Update maj.
+We do not handle them with the same standards:
+
+- We are very strict and meticulous when it comes to New modules.
+- For major updates, we have the exact same verifications than the New modules.
+- For minor updates, we verify the validator and we also have a Diff Tool to only see the modifications.
+
+Let's now give you more details about how we handle shit.
+
+### 2. Treatment of New and Major updates
+
+What's a "New" Module ? 
+Actually, it's a Zip. Your module is the product and since you can submit several zips per Product, let's use the right words.
+
+Simple: The very first time you submit a zip for your module to the marketplace. It's a "New" zip.
+Please, don't even try to submit your module if you didn't ever use the [Validator](https://validator.prestashop.com/).
+That tool is very powerful and useful and it will make everbody win a lot of time.
+
+After fixing every feedback from the Validator, here's a list of extra points we verify: 
+
+* The presence of DROP/ALTER of PrestaShop tables.
+    * It's highly forbidden to apply any changes on PrestaShop Core tables. It's very very dangerous and we don't allow any risk.
+
+* Except for [Payment Modules](https://github.com/PrestaShop/paymentexample/blob/master/paymentexample.php#L150), the use of iframes is STRICTLY FORBIDDEN.
+
+* Every hook. If any on them is empty, we decline the zip.
+    * There a other things we decline like loading a JS file in the whole backoffice when it's not necessary. Make sure to only load what you need, when you need it.
+
+* Missing index.php files in your directories ? We decline.
```

Comment:
Do we explain (maybe in another part of the doc) why these index.php files are required ? I guess yes, maybe we can provide the link in order to remind people of the importance of these files.

Reply:
Yes it's explained in another article. I'm adding a link for explanations